acoustid/data/account.py

```
@@ -82,28 +76,29 @@ def insert_account(conn, data):
         )
         .returning(schema.account.c.id, schema.account.c.apikey)
     )
-    id, api_key = conn.execute(insert_stmt).fetchone()
+    row = conn.execute(insert_stmt).fetchone()
+    if row is None:
+        raise Exception("Failed to insert account")
```

Ruff (0.8.2)

Create your own exception

(TRY002)